Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/viewmodel/CWAViewModel.kt

```diff
@@ -29,7 +29,13 @@ abstract class CWAViewModel constructor(
     fun launch(
         context: CoroutineContext = dispatcherProvider.Default,
         block: suspend CoroutineScope.() -> Unit
-    ): Job = viewModelScope.launch(context = context, block = block)
+    ) {
+        try {
+            viewModelScope.launch(context = context, block = block)
+        } catch (e: CancellationException) {
+            Timber.w(e, "launch()ed coroutine was canceled.")
```

Comment:
what's with the brackets? severity should be e in my opinion

Reply:
> what's with the brackets?

Wordplay on the functions name :grin: 

> severity should be e in my opinion

As argued by Roman in #1588 a cancelation is not an error but part of normal behavior. Reporting this as error would distract us from actual errors. It's still a warning and not a `.d` because we might want to look into what operations regularly run into this.